src/k8s-engine/lima.ts

```diff
@@ -1326,13 +1372,19 @@ export default class LimaBackend extends events.EventEmitter implements K8s.Kube
         }
 
         // We have no good estimate for the rest of the steps, go indeterminate.
-        timers.clearInterval(this.progressInterval);
```

Comment:
We can always clear the interval even if we never set it.

Reply:
Bump because comments were getting lost.

Reply:
> We can always clear the interval even if we never set it.

`lima.ts` line 1388 (removal on line 1341).
> timers.clearInterval(this.progressInterval);

Reply:
Done.